apps/ui/src/networks/offchain/api/index.ts

```diff
@@ -239,7 +239,10 @@ function formatSpace(
       : constants.EDITOR_VOTING_TYPES,
     min_voting_period: space.voting.period ?? 0,
     max_voting_period: space.voting.period ?? 0,
-    proposal_threshold: '1',
+    proposal_threshold:
+      String(validationParams.minScore) ||
+      String(validationParams.scoreThreshold) ||
```

Comment:
`scoreThreshold` is coming from passport right? i think it has no relation to voting power